.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
**Your project stills missing `ESLint` in your `Pull Request`.**
_Keep on mind if you added already `ESLint` setup files it's required to run the installations following the steps described on_ [this section](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Linters should appear like this:
![Screen Shot 2021-12-13 at 20 57 27](https://user-images.githubusercontent.com/22258274/145924943-2d8caeae-e1c0-48fc-9a81-dd6e636ef9a7.png)

#
Add your `ESLint` to your list:

![Screen Shot 2021-12-13 at 20 57 53](https://user-images.githubusercontent.com/22258274/145924983-577ce867-4395-4893-b970-360576487143.png)